cucumber/api/features/support/rest_helpers.rb

```diff
@@ -282,7 +282,10 @@ def current_user_api_key
 
   def current_user_credentials
     username = @current_user.login
-    token = Slosilo["authn:#{@current_user.account}"].signed_token(username)
+    # Configure Slosilo to produce valid access tokens
+    slosilo = Slosilo["authn:#{@current_user.account}"] ||= Slosilo::Key.new
```

Comment:
@amosmintzcyberark 
Can we add test to a case where `cucumber:user:alice` is not present at Slosilo leading us to  `Slosilo::Key.new`

Reply:
we can, but it's not related to the whoami, it's related to authentication. the whoami can't fail on authentication (it's a paradox)